packages/terra-form-select/src/shared/_Option.jsx

```diff
@@ -121,7 +121,7 @@ const Option = ({
   let label = display;
   // Allows VO to announce index of items
   if (SharedUtil.isMac() && index && totalOptions) {
-    label = `${display} (${index} ${ofText} ${totalOptions})`;
+    label = `${display}`;
```

Comment:
Isn't this affecting all terra-form-select variants since it's updating a shared file?